index.js

```diff
@@ -107,12 +109,23 @@ function fastifyView (fastify, opts, next) {
   })
 
   function getPage (page, extension) {
-    if (viewExt) {
-      return `${page}.${viewExt}`
-    } else if (includeViewExtension) {
-      return `${page}.${extension}`
+    const cacheId = `getPage-${page}-${extension}`
+    let result = lru.get(cacheId)
```

Comment:
Please use a difference/new cache for this, call it `pageLRU`

Reply:
Done

Reply:
Ohhh!!! sorry. It was my mistake, isn't it? Is it about the concatenated name of the variable name?

Reply:
You are using an LRU with mixed content. Instead, the content should be omogeneous. Instantiate a new lru cache and use that one for this purpose.

Reply:
> You are using an LRU with mixed content. Instead, the content should be omogeneous. Instantiate a new lru cache and use that one for this purpose.

Is it possible using the same `maxCache` option?
``` js
  const lru = HLRU(opts.maxCache || 100)
  const pageLRU = HLRU(opts.maxCache || 100)
```

Or what's the recommendation there?

Reply:
Like this is ok as pageLRU will not be big as it stores only a tiny string.